packages/dev-kit-italia/styles/index.scss

```diff
@@ -29,6 +29,7 @@
 @import 'bootstrap-italia/src/scss/components/header';
 @import 'bootstrap-italia/src/scss/components/headerslim';
 @import 'bootstrap-italia/src/scss/components/avatar';
+@import 'bootstrap-italia/src/scss/components/buttons';
```

Comment:
Se questo import ci serve solo per documentare i button che non sono web components, non lo metteri. Nell'uso reale del kit, direi che vogliamo che l'utente usi it-button e questi stili sono già in quello shadow dom.